edge-util/src/Microsoft.Azure.Devices.Edge.Util/PeriodicTask.cs

```diff
@@ -85,9 +89,22 @@ async Task DoWork()
                 {
                     try
                     {
-                        this.logger.LogInformation($"Starting periodic operation {this.operationName}...");
+                        // void Log(string message) => this.verbose ? this.logger.LogInformation(message) : this.logger.LogDebug(message);
```

Comment:
You didn't like the one-liner?

Reply:
It wasn't valid c#. I should remove it. I thought it would work, but it didn't for some reason.

Reply:
this syntax requires you to return a value :(